tasks/sanity-inspect-image.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sanity-inspect-image
+spec:
+  params:
+    - name: SOURCE_IMAGE
+      description: the fully qualified image name
+      default: quay.io/hongweiliu/skopeo-inspect-test:latest
+      type: string
+    - name: BASE
+      description: Inspecting base image for source image
+      default: 'false'
+      type: string
+  steps:
+  - name: inspect-image
+    image: quay.io/redhat-appstudio/hacbs-test:latest
+    script: |
+      IMAGE_INSPECT_FILE=$(workspaces.conftest-ws.path)/image_inspect.json
```

Comment:
I'm not sure if `conftest-ws` fits in this context, since we are not using conftest here at all, maybe sanity-ws makes more sense?

Reply:
Yes. `sanity-ws` makes sense.